modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/display/context/ViewNotificationTemplatesDisplayContext.java

```diff
@@ -76,17 +79,16 @@ public CreationMenu getCreationMenu() {
 			return creationMenu;
 		}
 
-		return creationMenu.addDropdownItem(
-			dropdownItem -> {
-				dropdownItem.setHref(
-					_getPortletURL(), "mvcRenderCommandName",
-					"/notification_templates/edit_notification_template",
-					"backURL", _notificationRequestHelper.getCurrentURL());
-				dropdownItem.setLabel(
-					LanguageUtil.get(
-						_notificationRequestHelper.getRequest(),
-						"add-notification-template"));
-			});
+		_addDropdownItem(
+			creationMenu, "email", NotificationConstants.TYPE_EMAIL);
+
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-162245"))) {
```

Comment:
Hi @MarinhoFeliphe, is this feature flag is correct? I think it should be the LPS of the current story: [LPS-162133](https://issues.liferay.com/browse/LPS-162133)

Reply:
@alinedoleron

my bad!, I changed to the right LPS

Thank you!